Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM golang as builder
+FROM golang:alpine as builder
 
 WORKDIR /go/src/github.com/paulfantom/periodic-labeler
 COPY . .
 
-RUN CGO_ENABLED=0 go build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags="-w -s"
 
-FROM gcr.io/distroless/static
+FROM scratch
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
```

Comment:
This is already included in `gcr.io/distroless/static` and that's why this base image was chosen.